lib/default-opts.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   registry: 'https://registry.npmjs.org/',
   timeout: 5 * 60 * 1000, // 5 minutes
   strictSSL: true,
-  noProxy: process.env.NOPROXY,
+  noProxy: process.env.NO_PROXY,
```

Comment:
This is a breaking change.  Folks currently relying on `NOPROXY` being used would no longer be able to do so.